pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -489,15 +489,21 @@ func (k *Kubernetes) ConfigEnvs(name string, service kobject.ServiceConfig) []ap
 // CreateKubernetesObjects generates a Kubernetes artifact for each input type service
 func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions) []runtime.Object {
 	var objects []runtime.Object
+	var replica int
+	if service.Replicas == 0 {
+		replica = opt.Replicas
```

Comment:
This doesn't make sense. We should check to see if the default `replicas` value has been changed, rather than checking for `0`.